build-tools/src/integTest/groovy/org/elasticsearch/gradle/test/YamlRestTestPluginFuncTest.groovy

```diff
@@ -26,14 +26,12 @@ class YamlRestTestPluginFuncTest extends AbstractGradleFuncTest {
         def result = gradleRunner("dependencies").build()
 
         then:
-        result.output.contains("""
+        normalized(result.output).contains("""
```

Comment:
Should we normalize the output once and assign it to local variable? Or is that overkill?

Reply:
Actually, we could could probably do the normalization already in the runner. I don't see where non normalized output would be preferable. Will look into this. 👍